### DISCUSSION TITLE
Fetching {incremental_value} for incremental load from different schema than target.

### DISCUSSION THREAD

**Discussion by rohitbatta** [slingdata-io/sling-cli#382] [Q&A]
I am trying to load the data from source (oracle) to target (sqlserver). I am using incremental load approach to fetch delta and using last_modified column to load the data. Once data is loaded to sqlserver schema "staging", i would transform that data to different schema and cleanup staging schema. 
For next run, there is no record in staging schema to calculate max_value for last_modified. Is there any option to pick the {incremant_value} dynamically from other schema from same database or picking max value from different table and pass it as {increment_value}

**Comment by flarco** [comment]
You can now do this with [hooks](https://docs.slingdata.io/concepts/hooks).
